### DISCUSSION TITLE
Set page size to A4 and add margins

### DISCUSSION THREAD

**Discussion by bwiti-me** [pymupdf/PyMuPDF#3436] [Looking for help] [answered]
I'm using the script 'convert.py' that I found on this site to convert files to pdf. It works out of the box, but I'd like to enahance it with two functionalities:
- set paper size to A4
- add a 1 inch margin top, bottom, left and right
I could of course convert the initially created pdf to A4, but if possible I'd like to flow the contents of the epub (or whatever) over the A4, instead of re-scaling.

I hope this is clear and appreciate any help provided.

bwiti-me

**Comment by JorjMcKie** [comment, marked as answer]
You can layout a reflowable document with the desired page format - either already at open or later via `doc.layout(width=595, height=842)`.
Several filetypes are reflowable, among them EPUB.

**Reply by bwiti-me** [reply]
Hey, thanks. I tried that and it works. I also found a way to create margins with show_pdf_page. So all is good.
